neutrons/reflectivity_ui
# Remove ApplicationConfiguration

The class ApplicationConfiguration was used previously to manage Mantid dependencies. Now, we use conda environments to ensure QuickNXS has the correct dependencies, like Mantid. I suggest we remove this class.

### To test

Build using `python -m pip install -e .` and check that you can run the executable `bin/quicknxs2`.
